R/produce_nrs_extract_tests.R

```diff
@@ -0,0 +1,43 @@
+#' Source Extract Tests
+#'
+#' @description Produce a set of tests which can be used by the death (NRS) extract
+#'
+#' This will produce counts of various demographics.
+#' It will also produce various summary statistics for episode date variables.
+#'
+#' @param data new or old data for testing summary flags
+#' (data is from [get_source_extract_path()])
+#' @param max_min_vars variables used when selecting 'min-max' from [calculate_measures()]
+#'
+#' @return a dataframe with a count of each flag
+#' from [calculate_measures()]
+#' @export
+#'
+#' @family extract test functions
+#' @seealso [calculate_measures()]
+produce_nrs_extract_tests <- function(data,
+                                      max_min_vars = c(
+                                        "record_keydate1", "record_keydate2"
+                                      )) {
```

Comment:
```suggestion
produce_source_nrs_tests <- function(data) {
```
Name suggestion and remove parameter for max_min_vars. If you have a look at PIS tests, James suggested that we should hardcode these variables instead of setting a parameter because they have their own test function

Reply:
Applied suggestion